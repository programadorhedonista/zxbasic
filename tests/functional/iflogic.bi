
#define A 1
#define B 2

#if A == 1 || B == 1
PRINT "Or works"
#endif

#if A == 1 && B == 2
PRINT "And works"
#endif

#if (A == 1 && B == 1) || B == 2
PRINT "Parenthesis works"
#endif

#if A == 1 && B != 2
PRINT "this should not happen!"
#endif
