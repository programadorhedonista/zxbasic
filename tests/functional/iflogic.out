#line 1 "iflogic.bi"

#line 3 "iflogic.bi"
#line 4 "iflogic.bi"


PRINT "Or works"
#line 8 "iflogic.bi"


PRINT "And works"
#line 12 "iflogic.bi"


PRINT "Parenthesis works"
#line 16 "iflogic.bi"

#line 20 "iflogic.bi"
